packages/components/src/draggable/index.native.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import Animated, { useSharedValue } from 'react-native-reanimated';
+
+/**
+ * Draggable component
+ *
+ * @param {Object}                                      props                       Component props.
+ * @param {JSX.Element}                                 props.children              Children to be rendered.
+ * @param {?number}                                     props.maxDistance           Maximum distance, that defines how far the finger is allowed to travel during a long press gesture.
+ * @param {?number}                                     props.minDuration           Minimum time, that a finger must remain pressed on the corresponding view.
+ * @param {Function}                                    props.onDragEnd             Callback when dragging happens over an element.
+ * @param {Function}                                    props.onDragOver            Callback when dragging ends.
+ * @param {Function}                                    props.onDragStart           Callback when dragging stars.
+ * @param {import('react-native-reanimated').StyleProp} props.wrapperAnimatedStyles Animated styles for the wrapper component.
```

Comment:
If this prop is optional, we could note it in the documentation:
```suggestion
 * @param {import('react-native-reanimated').StyleProp} [props.wrapperAnimatedStyles] Animated styles for the wrapper component.
```